Resources/views/forms/fields/default-field.html.twig

```diff
@@ -46,53 +40,62 @@
                         </label>
                     </div>
                 {% endblock %}
+                {% block width %}
+                    <div class="grid-col-3 m-top-20 floating form-group pull-right">
+                        <div id="width<~=index~>"
+                             class="preview-update trigger-save-button"
+                             data-aura-component="select@husky"
+                             data-aura-data='{{ widths|json_encode() }}'
+                             data-aura-multiple-select="false"
+                             data-aura-instance-name="width"
+                             data-aura-default-value='{"id":"full","name":"sulu_form.width.full"}'
+                             data-type="husky-select"
+                             data-form="true"
+                             data-mapper-return-value="id"
+                             data-mapper-property="width">
+                        </div>
+
+                        <div class="block-content" data-sort-mode-id="{{ blockId }}width<~=index~>"></div>
+
+                        <div class="input-description"></div>
+                    </div>
+                    <div class="grid-col-1 m-top-25 floating form-group pull-right">
+                        <label for="width<~=index~>"
+                               class="pointer"
+                               title="width">
+                            <%= translate('sulu_form.width') %>
+                        </label>
+                    </div>
+                {% endblock %}
             </div>
 
             <div class="block block-{{ type }} grid-row small">
-                {% block title_width %}
+                {% block title_default %}
```

Comment:
This block does not contain the `default` block. Because of the issue #3 I would call it `title_short_title` .